package/MDAnalysis/coordinates/base.py

```diff
@@ -1309,28 +1310,37 @@ def __repr__(self):
                 ))
                 
     def add_auxiliary(self, auxdata, auxname, **kwargs):
+        """Add auxiliary data to be read alongside trajectory and match to 
+        current timestep.
```

Comment:
This is a function actual final users will use (not only dev or power-users). It will require more documentations:

* what are auxiliary data?
* when do we need them?
* how do I use the method, and how do I access the data? With examples.